src/setup/setup-strategy-cards.js

```diff
@@ -95,4 +95,6 @@ class SetupStrategyCards extends AbstractSetup {
     }
 }
 
-module.exports = { SetupStrategyCards };
+// TODO: STRATEGY_CARDS exported for end of round cleanup remove it once
+// strategy cards have a better defined home/mat.
+module.exports = { SetupStrategyCards, STRATEGY_CARDS };
```

Comment:
We should probably export a function to get the position/rotation b/c that's going to change when the strategy card mat is added.

Reply:
I made the other changes, but I think I will move this to a different task/pull request